lib/servers/iXonServer/qt4reactor.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2001-2008 Twisted Matrix Laboratories.
```

Comment:
This appears to be an old version (<1.0) of qt4reactor.py (current is 1.6).  

https://pypi.python.org/pypi/qt4reactor